website/docs/components/inline-edit/inline-edit-code.md

```diff
@@ -29,7 +29,7 @@ As far as `<InlineEdit />` is flexible, it could be used with almost any childre
 
 ## Pseudo network interaction
 
-Component stays simple to provide you a way to handle edited values in any needed way.
+Component stays simple to provide you a way to handle edited values in any way you need them.
```

Comment:
Would be nice to rephrase the "component stays simple" part, because 1) vale complains, and 2) it's not even clear what we mean here, I personally don't understand

Reply:
That's my question to guys actually 😁